gno.land/pkg/gnoland/param.go

```diff
@@ -0,0 +1,121 @@
+package gnoland
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/gnolang/gno/tm2/pkg/sdk"
+	"github.com/gnolang/gno/tm2/pkg/sdk/params"
+)
+
+type Param struct {
+	key   string
+	kind  string
+	value interface{}
+}
+
+func (p Param) Verify() error {
+	// XXX: validate
```

Comment:
Leftover?

Reply:
People may want to add early validation here. It will be validated later in the runtime anyway. I'm okay with keeping the comment, but I can remove it if you prefer.